.github/workflows-source/ci-main.yml

```diff
@@ -117,6 +117,15 @@ jobs:
           if-no-files-found: error
           path: |
             bazel-targets
+      - name: Upload bazel-bep
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ github.job }}-bep
+          retention-days: 90
```

Comment:
why does the retention period need to be so long?

Reply:
Benefit is being able to repopulate the database if we lose the data. The size of artifacts is also not big. Do you see concern here?

Reply:
Yep totally fine! Though a bit unexpected; I had to double check the default value